models/models.go

```diff
@@ -10,10 +10,10 @@ type Price int64
 
 //User represents the users making the bid
 type User struct {
-	UserID   string `json:"user_id, string"`
+	UserID   string
```

Comment:
Can you please explain the reason behind this change to UserID, Password, Points?